examples/albert/arguments.py

```diff
@@ -138,9 +138,14 @@ class AlbertTrainingArguments(TrainingArguments):
     fp16: bool = True
     fp16_opt_level: str = "O2"
     do_train: bool = True
+    do_eval: bool = False
 
+    logging_dir: str = "logs"
+    output_dir: str = "outputs"
     logging_steps: int = 100
+    logging_first_step: bool = True
+    overwrite_output_dir: bool = True
+
     save_total_limit: int = 2
     save_steps: int = 500
-
-    output_dir: str = "outputs"
+    max_steps: int = 10 ** 30
```

Comment:
```suggestion
```

Let's do of the options below:

(1) remove it (it's inf by default AFAIU)
(2) set it to a reasonable value
(3) add a comment that you mean `inf` by writing `10 ** 30` (now, it's not 100% clear what this number means, someone may think that you really need that much microsteps)

Reply:
went with a 3